Week3/homework/1-step3.js

```diff
@@ -1,9 +1,11 @@
 'use strict';
 
 function foo(func) {
-  // What to do here? 
-}
+  const x = 12;
 
+  console.log('My address is :' + x);
+  func();
```

Comment:
The value that is returned from calling `func()` should be returned from the function. e.g.
```
return func()
```
`foo` should know nothing about the function it is calling, and so can't make the assumption that the return value is not needed somewhere else.